### ISSUE TITLE
⚡ — Add icon for Gluetun ;-)

### ISSUE THREAD

**Issue by cerealconyogurt** [walkxcode/dashboard-icons#300]
### Type

Icon request

### Info

Hello! please add icon for Gluetun

[github.com/qdm12/gluetun](https://github.com/qdm12/gluetun)

Thanks

### Icon(s)

![gluetun](https://user-images.githubusercontent.com/49502224/222930445-a4bef422-ea53-4412-b1a7-1aeae385e1bd.png)

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/gluetun.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/svg/gluetun.svg
